src/lib/reasoners/bitv.ml

```diff
@@ -587,11 +587,19 @@ module Shostak(X : ALIEN) = struct
        Ensures: there are no duplicates in the result (in particular, [x = y]
                 and [y = x] cannot be both present)
        Ensures: there are no trivial equalities [x = x] in the result. *)
+
+    let equal_pair_simple_term ((a1, b1) as p1) ((a2, b2) as p2) =
+      if p1 == p2 then true
```

Comment:
This is never going to be true given the way we are using this function, no need for a shortcut.